packages/tools/src/tools/PanTool.ts

```diff
@@ -29,28 +29,38 @@ class PanTool extends BaseTool {
   _dragCallback(evt: EventTypes.InteractionEventType) {
     const { element, deltaPoints } = evt.detail;
     const enabledElement = getEnabledElement(element);
+    const viewport = enabledElement.viewport;
 
-    const deltaPointsWorld = deltaPoints.world;
-    const camera = enabledElement.viewport.getCamera();
-    const { focalPoint, position } = camera;
-
-    const updatedPosition = <Types.Point3>[
-      position[0] - deltaPointsWorld[0],
-      position[1] - deltaPointsWorld[1],
-      position[2] - deltaPointsWorld[2],
-    ];
-
-    const updatedFocalPoint = <Types.Point3>[
-      focalPoint[0] - deltaPointsWorld[0],
-      focalPoint[1] - deltaPointsWorld[1],
-      focalPoint[2] - deltaPointsWorld[2],
-    ];
-
-    enabledElement.viewport.setCamera({
-      focalPoint: updatedFocalPoint,
-      position: updatedPosition,
-    });
-    enabledElement.viewport.render();
+    if (viewport instanceof VideoViewport) {
+      const videoViewport: VideoViewport = <VideoViewport>viewport;
+
+      const { pan } = videoViewport.getCamera();
+
+      videoViewport.setCamera({
+        pan: [pan[0] + deltaPoints.world[0], pan[1] + deltaPoints.world[1]],
+      });
+    } else {
```

Comment:
Not following the ICamera leads to this side effect, which will occur during Phase 2 for annotation and segmentations.